src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -71,7 +73,8 @@ public void setVisibility(Visibility visibility) {
 
     @Override
     public Player removePlayer(String token) {
-        return playerMap.remove(token);
+        Player player = playerMap.remove(token);
+        return player;
```

Comment:
why the change?
Seems unnecessarily verbose to me at first sight

Reply:
Probably during merge, fixed.